test/OpenTelemetry.Extensions.Docker.Tests/OpenTelemetry.Extensions.Docker.Tests.csproj

```diff
@@ -3,7 +3,8 @@
   <PropertyGroup>
     <Description>Unit test project for Docker Detector for OpenTelemetry</Description>
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
I think that you should merge this line and the line before to
`<TargetFrameworks>net6.0;netcoreapp3.1</TargetFrameworks>`

Reply:
@Kielek, I have addressed the comments and updated the PR